src/components/modals/medication/medicationModalInfo.ts

```diff
@@ -1,7 +1,14 @@
-export interface AddMedicationInfo {
+import { DayOfWeek } from "@/lib/consts";
+
+export interface MedicationModalInfo {
+  /**
+   * If the Model ID is required, in Edit Modal
+   */
+  _id?: string;
+
   general: {
     /** Type of medication */
-    form: "Pill" | "Injection";
+    form: "tablet" | "liquid" | "injection";
```

Comment:
```suggestion
    form: "tablet" | "liquid" | "injection";
```

In the actual frontend, it specifies pill, syrup, and shot. Can we make the datatypes consistent with those strings?